test/parallel/test-process-versions.js

```diff
@@ -33,6 +33,20 @@ assert(/^\d+\.\d+\.\d+(?:\.\d+)?-node\.\d+(?: \(candidate\))?$/
   .test(process.versions.v8));
 assert(/^\d+$/.test(process.versions.modules));
 
+if (common.hasCrypto) {
+  assert(/^\d+\.\d+\.\d+[a-z]?$/.test(process.versions.openssl));
+}
```

Comment:
It may be helpful to include a comment along with each one that illustrates what the expected version looks like.